src/utils/construct-input-object.test.ts

```diff
@@ -37,6 +37,32 @@ describe('#constructInputObject()', () => {
 		});
 	});
 
+	it('Handles passthrough inputs too', async () => {
+		process.argv = [
+			'/path/to/node',
+			path.join(testProjectsPath, 'pizza-ordering', 'cli', 'entry.js'),
+			'order',
+			'to-go',
+			'--',
+			'--pass-through-flag',
+			'pass-through-option=value',
+			'pass-through-data',
+		];
+
+		expect(await constructInputObject()).toStrictEqual({
+			command: 'order to-go',
+			data: undefined,
+			options: {
+				'delivery-zip-code': undefined,
+				test: undefined,
+			},
+			flags: {
+				quiet: false,
+			},
+			passThroughArgs: ['--pass-through-flag', 'pass-through-option=value', 'pass-through-data'],
+		});
+	});
+
```

Comment:
Should this test be added to this PR? Unless I'm missing something, it seems unrelated.

Reply:
Without this test being added, the changes appear to expose some code to a lack of test coverage, so I added this to help keep the coverage score up.